posts/2021-03-13-microprofile-config-2.0.adoc

```diff
@@ -240,7 +254,7 @@ The behaviour for "empty" and "special" config property values has been updated:
 * A value is considered to be "empty" if the link:https://download.eclipse.org/microprofile/microprofile-config-2.0/apidocs/org/eclipse/microprofile/config/spi/Converter.html[Converter] being used considers it to be "empty". For example: 
 ** All Converters consider `""`, the empty String, to be empty.
 ** The built-in Converter for String[] considers `","` to be empty (because it is "special"). 
-* Retrieving these "empty" values "natively" will now throw a `NoSuchElementException`, e.g. for the values defined in the <<example-ConfigSource2, example ConfigSource v2>>:
+* From MicroProfile Config 2.0, these "empty" values are no longer valid. Retrieving the values "natively" will now throw a `NoSuchElementException`. E.g. for the values defined in the <<example-ConfigSource2, example ConfigSource v2>>:
```

Comment:
```suggestion
* From MicroProfile Config 2.0, these "empty" values are no longer valid. Retrieving the values directly using the Config API will now throw a `NoSuchElementException`. E.g. for the values defined in the <<example-ConfigSource2, example ConfigSource v2>>:
```

Reply:
This isn't what I meant by "natively". I meant, retrieving the value without using default values or Optionals:
https://download.eclipse.org/microprofile/microprofile-config-2.0/apidocs/org/eclipse/microprofile/config/inject/ConfigProperty.html

But perhaps this should be explained

Reply:
I've updated this. What'cha think?